diff -urN Quota-1.4.1/linuxapi.c Quota-1.4.1-new/linuxapi.c
--- Quota-1.4.1/linuxapi.c	Sun Sep  2 20:36:16 2001
+++ Quota-1.4.1-new/linuxapi.c	Mon Feb 25 17:37:02 2002
@@ -18,11 +18,12 @@
 #define Q2_GETSTATS  0x1100
 
 
-#if defined(INITQFNAMES) && (Q_GETQUOTA==0x0300) && (Q_GETSTATS==0x0800)
+#if 0 && defined(INITQFNAMES) && (Q_GETQUOTA==0x0300) && (Q_GETSTATS==0x0800)
 /* this is an early version of the quota patch which did not have the
 ** version element in the stats struct. Hence it's impossible to detect
-** at runtime with a sufficient degree of certainity. */
-# define LINUX_API_V2_ONLY
+** at runtime with a sufficient degree of certainity.
+** DISABLED by bartowl@pld.org.pl due to bad version detection */
+#define LINUX_API_V2_ONLY
 #endif
 
 
